.github/workflows/make-swagger.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: docker build -f docker.local/build.base/Dockerfile.build_base . -t zchain_build_base
     
       - name: build swagger image
-        run: docker build -f MakeswaggerDockerFile/Dockerfile . -t make_swagger
+        run: docker build -f docker.local/build.swagger/MakeswaggerDockerFile/Dockerfile . -t make_swagger
```

Comment:
Please do not create duplicate Dockerfile, we can just one the docker.local/build.swagger/Dockerfile, please check the docker.local/bin/test.swagger.sh to see how to use it.

Reply:
@peterlimg  earlier checked this but  it popped up error that's why i used different way

Reply:
@Manali-Jain-squareops please check [deploy-swagger](https://github.com/0chain/0chain/commit/bfcc0793716453c364461b761ce68d5c75aeee96) branch and adjust it here, or just redo the PR on top of that branch.